cloudformation/elasticsearch.yaml

```diff
@@ -81,10 +99,23 @@ Resources:
     Properties:
       IdentityPoolId: !Ref KibanaIdentityPool
       Roles:
-        'authenticated': !GetAtt AdminKibanaAccessRole.Arn
+        authenticated: !GetAtt AdminKibanaAccessRole.Arn
+  KibanaDevUser:
+    Type: AWS::Cognito::UserPoolUser
+    Condition: isDev
+    Properties:
+      DesiredDeliveryMediums:
+        - EMAIL
+      UserAttributes:
+        - Name: email
+          Value: !Ref KibanaUserEmail
+      Username: !Ref KibanaUserEmail
+      UserPoolId: !Ref KibanaUserPool
+
   ElasticSearchDomain:
     Type: AWS::Elasticsearch::Domain
     Properties:
+      DomainName: fhir-domain

```

Comment:
Why are we setting this? If we are we will need to add more dimensionality (e.g. adding stage)

Reply:
Agreed, I was just seeing how adding a name to the ES cluster would affect how the User Pool domain. Will remove